BlueprintUICommonControls/Sources/AccessibilityBlocker.swift

```diff
@@ -33,7 +33,7 @@ public struct AccessibilityBlocker: Element {
 
     public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
         UIView.describe { config in
-            config[\.isAccessibilityElement] = !isBlocking
```

Comment:
This UIView should never be an accessibilityElement because it's always wrapping another element. 
if this gets set to true (when isBlocking is false) it also blocks the accessibility because voiceover never asks for its children.